src/Internal/Services/PaymentService.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Class PaymentService
+ *
+ * @package WooPayments
+ */
+
+namespace WooPayments\Internal\Services;
+
+/**
+ * Processing payment service.
+ */
+class PaymentService {
```

Comment:
I was about to ask what you'd think about `PaymentProcessingService` rather than `PaymentService`, then I saw the doc block. Let's make the service specific to _processing_ payments.

Reply:
👍 Addressed 6192259